src/Backups/BackupSettings.cpp

```diff
@@ -72,6 +71,7 @@ namespace
     M(String, host_id) \
     M(String, coordination_zk_path) \
     M(OptionalUUID, backup_uuid)
+    /// M(Int64, compression_level)
```

Comment:
This is deeply in the defines. It can be fixed by checking types or cast for any setting or by using Int64 in BackupSettings.h but all of this looks worse. Thoughts?

Reply:
Another option is to turn the warning off around this code using a pragma.

Reply:
I thought about this, but it is too tempting to have such example in the code :)

Reply:
Fine with me then.